.github/workflows/ci.yaml

```diff
@@ -74,13 +74,21 @@ jobs:
           file: tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/Dockerfile
 
   run-docker-tests-on-garm:
-    needs: publish-kata-deploy-payload-amd64
+    needs: build-kata-static-tarball-amd64
```

Comment:
It seems that this change belongs to next patch, and the change from the next patch should go...